src/components/FamilyManifestModal/FamilyManifestModal.js

```diff
@@ -86,68 +88,69 @@ const enhance = compose(
         }
       }
     `,
-    variables: {
-      sqon: {
-        op: 'and',
-        content: [
-          {
-            op: 'in',
-            content: { field: 'participants.kf_id', value: familyMemberIds },
-          },
-          ...(sqon ? [{ op: 'not', content: [sqon] }] : []),
-        ],
-      },
-    },
-  })),
-  withProps(({ familyMemberWithoutParticipantIdAggregation: { data } }) => ({
-    familyMemberWithoutParticipantIds: (
-      get(data, 'file.aggregations.participants__family__family_members__kf_id.buckets') || []
-    ).map(b => b.key),
-  })),
-  withProps(({ familyMemberIds, familyMemberWithoutParticipantIds }) => ({
-    finalFamilyMemberIds: familyMemberWithoutParticipantIds.length
-      ? familyMemberWithoutParticipantIds
-      : familyMemberIds,
+    variables: { sqon },
   })),
+  withProps(({ participantIdsAggregation: { data }, familyMemberIds }) => {
+    const participantIds = (get(data, 'file.aggregations.participants__kf_id.buckets') || []).map(
+      b => b.key,
+    );
+    return {
+      participantIds,
+      familyMembersWithoutParticipantIds: familyMemberIds.filter(
+        id => !participantIds.some(pId => id === pId),
+      ),
```

Comment:
this is always going to run `O(m*n)`, I _think_ lodash `difference` has an optimization for larger array sizes, should be able to just do `difference(participantIds, familyMemberIds)`

Reply:
haha it does read better too :D I should really start using more of lodash lodash